py/vbcl/__init__.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+#
+# VBCL parser
+#
+# Andrew Bernard 2017
+
+import re
+
+# compile the match patterns
+comment = re.compile(r"^#")
+nv_pair = re.compile(r"^(.*): (.*)$")
+long_text_start = re.compile(r"^(.*): <")
+long_text_end = re.compile(r"^  >")
+list_items_start = re.compile(r"^(.*): \[")
+list_items_end = re.compile(r"^  \]")
+
+defaults = {
+    'name': 'NN',
+    'display-name': 'NN',
+    'short-description': 'No short description available',
+    'description': 'No description available',
+    'dependencies': [],
+    'oll-core': '0.0.0',
+    'maintainers': [],
+    'version': '0.0.0',
+    'license': 'None'
+}
+
+
+def set_defaults(d):
+    """Ensure mandatory properties are set to 'empty' values."""
+    for key in defaults:
+        d[key] = d.get(key, defaults[key])
+    return d
+
+
+def parse(lines):
+    """Returns a dictionary corresponding to a parsed VBCL string list."""
+    d = dict()
+    it = iter(lines)
+
+    try:
+        while True:
+            line = next(it)
+            # comments - discard
+            if comment.search(line):
+                continue
+            else:
+                # long text
+                m = long_text_start.search(line)
+                if m:
+                    text = str()
+                    while True:
+                        line = next(it)
+                        if long_text_end.search(line):
+                            d[m.group(1)] = text.strip('\n')
+                            break
+                        text += (line.strip(' '))
```

Comment:
Shouldn't this read `line.strip()`?

I think it's not necessary to limit this to `SPACE` and rather accept generic whitespace (i.e. including tabs)